warehouse/banners/models.py

```diff
@@ -21,10 +21,18 @@
 class Banner(db.Model):
     __tablename__ = "banners"
     __repr__ = make_repr("text")
+    DEFAULT_FA_ICON = "fa-comment-alt"
 
+    # internal name
     name = Column(String, nullable=False)
+
+    # banner display configuration
     text = Column(Text, nullable=False)
     link_url = Column(URLType, nullable=False)
+    link_label = Column(String, nullable=False)
```

Comment:
Would a default link label make sense as well ?

Reply:
Yes, it makes sense. Maybe just "Open" as the default?

Reply:
Hm, not fond of Open (but not a native English speaker). Was this the CTA usually used on previous campaigns ? (Otherwise, maybe "More" / "See more" / "Learn more")

Reply:
I'll go with See more then :+1:

Reply:
Fixed by af6341bccd1fb53df2d0e6548c0267938a9847dd